app/src/modules/sparql.js

```diff
@@ -17,24 +20,22 @@ async function querySparql (query, endpoint = SPARQL_ENDPOINT) {
     .catch((err) => console.log(err))
 }
 
-export { querySparql }
-
-export function parseSPARQL (response) {
+function parseSparql (response) {
   const queryResults = []
-  for (let i = 0; i < response.results.bindings.length; i++) {
-    const newObject = {}
-    const row = response.results.bindings[i]
-    for (const variable of response.head.vars) {
-      if (row !== undefined && row[variable] !== undefined && row[variable]) {
-        if (!isNaN(row[variable].value)) {
-          newObject[variable] = +row[variable].value
-        } else {
-          newObject[variable] = row[variable].value
+  if (response) {
```

Comment:
It may be better to just fail if there is an invalid response.  Doing nothing in the event of a bad response could make debugging more difficult.
(I realize that this is my code here)

Reply:
@aswallace would you indicate this in the new TD ticket so it can be part of that fix.

Reply:
Indicated in #125